tools/toit.toit

```diff
@@ -600,7 +600,8 @@ run-lsp-server invocation/cli.Invocation:
   toitc-cmd := [tool-path sdk-dir "toit.compile"]
   if toitc-cmd.size != 1: throw "Unexpected toitc command: $toitc-cmd"
   // We are not using the cli's Ui class, as it might print on stdout.
-  print-on-stderr_ "Using $toitc-cmd.first"
+  if invocation.cli.ui.level <= cli.Ui.VERBOSE-LEVEL:
```

Comment:
This feels a bit counterintuitive to me. I would think VERBOSE is a relatively high level with DEBUG being the only one higher. I see that the levels are negative, but that just adds to the confusion. 

Maybe we shouldn't expose the level like this? Having accessors like `cli.ui.is-verbose` would make it easier to reason about.